src/workflow/trinary_input.c

```diff
@@ -60,14 +65,15 @@ static void _cancel(void* param)
 }
 
 workflow_trinary_input_result_t workflow_trinary_input_wordlist(
+    size_t word_idx,
     const char* title,
```

Comment:
since you have the word index now, might as well get rid of the `title` arg and create it here, inside `workflow_trinary_input_wordlist`.

Reply:
Nice idea, done.